Sofa/framework/Core/src/sofa/core/behavior/BaseAnimationLoop.cpp

```diff
@@ -29,13 +29,20 @@ namespace sofa::core::behavior
 {
 
 BaseAnimationLoop::BaseAnimationLoop()
-    : m_resetTime(0.)
+    : l_node(initLink("targetNode","Link to the scene's node where the rendering will take place"))
```

Comment:
I am not sure "rendering" is the right term

Reply:
indeed. Thanks